### DISCUSSION TITLE
Possibility to Connect an OPC DA Server with this opcua-asyncio libraries

### DISCUSSION THREAD

**Discussion by dibyendumca** [FreeOpcUa/opcua-asyncio#1138] [Q&A]
Hello,

Does anyone have any idea, whether IS IT POSSIBLE TO CONNECT WITH AN **OPC DA SERVER** with this OPCUA-ASYNCIO libraries?

**Comment by AndreasHeine** [comment]
this is a "OPC UA" Library not "OPC Classic"!

**Reply by AndreasHeine** [reply]
you can use "pywin32" library

**Comment by harriv** [comment]
You need a gateway because OPC UA is not compatible with OPC DA. For example this: https://www.prosysopc.com/products/opc-ua-gateway/
